CONTRIBUTING.md

```diff
@@ -1,16 +1,21 @@
-# CONTRIBUTION GUIDELINES
+CONTRIBUTION GUIDELINES
+======
 
-## Before contributing
+Before contributing
+------
 
 Welcome to [TheAlgorithms/Go](https://github.com/TheAlgorithms/Go)! Before submitting pull requests, please make sure that you have **read the whole guidelines**. If you have any doubts about this contribution guide, please open [an issue](https://github.com/TheAlgorithms/Go/issues/new/choose) or ask in our [Discord server](https://discord.gg/c7MnfGFGa6) / [Gitter](https://gitter.im/TheAlgorithms), and clearly state your concerns.
 
-## Contributing
+Contributing
+------
```

Comment:
Please use # for headings, it's clearer IMO

Reply:
agree. but i want to keep the lines for clearer separation.

Reply:
level 3 and 4 heading is not obvious by default.

Reply:
adjusted